app/components/Navigation.tsx

```diff
@@ -0,0 +1,26 @@
+import Link from "next/link";
+
+export default function Navigation() {
+  return (
+    <nav className="bg-gray-200 p-4 rounded-md shadow-md border border-slate-500 mb-8">
+      <ul className="flex space-x-6">
+        <li>
+          <Link
+            href="/"
+            className="text-gray-800 font-semibold bg-gray-300 hover:bg-gray-400 hover:text-gray-900 py-2 px-4 rounded-md transition-colors duration-300"
+          >
+            To do list
+          </Link>
+        </li>
+        <li>
+          <Link
+            href="/AddCard"
```

Comment:
don't use different types of urls for example here you have CamelCase url, but next file it's /card-info/${information.id}
the standard is kebab case, so I suggest sticking to it, this applies to everything else, try to be consistent with your code.